packages/modules/internal_chargepoint_handler/clients.py

```diff
@@ -67,7 +73,7 @@ def find_meter_client(meters: List[meter_config], client: Union[ModbusSerialClie
                 try:
                     if meter_client.get_voltages()[0] > 200:
                         with ModifyLoglevelContext(log, logging.DEBUG):
-                            log.debug("Verbauter Zähler: "+str(meter_type)+" mit Modbus-ID: "+str(modbus_id))
+                            log.error("Verbauter Zähler: "+str(meter_type)+" mit Modbus-ID: "+str(modbus_id))
```

Comment:
```suggestion
                            log.debug("Verbauter Zähler: "+str(meter_type)+" mit Modbus-ID: "+str(modbus_id))
```
Die Anpassung ist nicht erforderlich, da der Loglevel in der Zeile darüber vom Kontexthandler bereits auf "debug" gesetzt wird. Die Ausgabe wird also bereits immer in das Log geschrieben.